src/lib/db.js

```diff
@@ -760,6 +760,44 @@ export function searchStaffQuery(queries, min, max) {
   });
 }
 
+export function searchScoredQuery(queries, min, max) {
```

Comment:
We can use a cool async function here and use async await!

Reply:
Then we can also do Promise.all that I didn't know about when I originally wrote our DB queries :), no more `queriesReturned`

Reply:
So it should instead be something like `const results = await Promise.all(queries.map(query => database. ...); return results` or something like that